openapi.yaml

```
@@ -799,6 +817,8 @@ components:
               enum:
                 - BEFORE_OR_SAME
     cq-days-before-or-never:
+      description: "Used within CONCEPT_QUERY to describe relative temporal relationships between complex events.
```

```suggestion
      description: "Used within CONCEPT_QUERY to describe relative temporal relationships between complex event arrangements.
```
Damit meine ich das *complex event arrangements* eine andere Query sind.